wiki/Tournaments/oPHN/2019_4K/en.md

```diff
@@ -46,10 +50,10 @@ The **osu! Philippines Nationals 2019 (osu!mania 4K)** (***PHN4K 2019***) is a o
 ## Links
 
 - [Discussion thread](https://osu.ppy.sh/community/forums/topics/902235)
-- [YouTube](https://www.youtube.com/channel/UCkV4V1OVO6gOGxtH51glH0Q/live)
-- [Twitch](https://www.twitch.tv/osuliveph)
-- [Challonge](https://challonge.com/oPHN4K19)
-- [osu! Philippines Tournament Discord server](https://discord.gg/9ZUMmwS)
+- [YouTube](https://www.youtube.com/channel/UCkV4V1OVO6gOGxtH51glH0Q/live "YouTube")
+- [Twitch](https://www.twitch.tv/osuliveph "Twitch")
+- [Challonge](https://challonge.com/oPHN4K19 "Challonge")
+- [osu! Philippines Tournament Discord server](https://discord.gg/9ZUMmwS "Discord")
```

Comment:
we don't use tooltips for links on the wiki like this. it's also redundant here

Reply:
it seems already updated this part.